src/app/home-page/home-page.controller.js

```diff
@@ -10,14 +11,17 @@ export default class HomePageController {
 
     search() {
         this.latestSearchesUtil.addCity(this.city);
-        //Will do this call on the search page
-        //this.openWeatherMaps.getFiveDayForecastCity(this.city);
+        this.openWeatherMaps.getCurrentForecastCity(this.city);
+        this.openWeatherMaps.getFiveDayForecastCity(this.city);
+        this.$state.go('search');
     }
 
     clickCity(city) {
-        //Will do this call on the search page
-        //this.openWeatherMaps.getFiveDayForecastCity(city);
+        this.latestSearchesUtil.addCity(city);
```

Comment:
this method does exactly the same as `search()`, avoid write redundant lines of code, you can delete this method, or just call the other

Reply:
Noticed on next task, Ill solve it here, havent done it yet on next one